src/lib/tames.ts

```diff
@@ -113,6 +115,86 @@ export const seaMonkeySpells: SeaMonkeySpell[] = [
 	}
 ];
 
+export interface SpecialLoot {
+	monster: number[];
+	specialItems: {
+		item: number[];
+		chance: number[];
+	}[];
+}
+
+export const specialLoot: SpecialLoot[] = [
+	{
+		monster: [Monsters.PhantomMuspah.id],
+		specialItems: [
+			{
+				item: [itemID('Charged ice')],
+				chance: [20]
+			}
+		]
+	},
+	{
+		monster: [Monsters.DukeSucellus.id, Monsters.AwakenedDukeSucellus.id],
+		specialItems: [
+			{
+				item: [itemID('Frozen tablet')],
+				chance: [25]
+			}
+		]
+	},
+	{
+		monster: [Monsters.Vardorvis.id, Monsters.AwakenedVardorvis.id],
+		specialItems: [
+			{
+				item: [itemID('Strangled tablet')],
+				chance: [25]
+			}
+		]
+	},
+	{
+		monster: [Monsters.TheWhisperer.id, Monsters.AwakenedTheWhisperer.id],
+		specialItems: [
+			{
+				item: [itemID('Sirenic tablet')],
+				chance: [25]
+			}
+		]
+	},
+	{
+		monster: [Monsters.TheLeviathan.id, Monsters.AwakenedTheLeviathan.id],
+		specialItems: [
+			{
+				item: [itemID('Scarred tablet')],
+				chance: [25]
+			}
+		]
+	},
+	{
+		monster: [BSOMonsters.VladimirDrakan.id],
+		specialItems: [
+			{
+				item: [
+					itemID('Vampyre hunter boots'),
+					itemID('Vampyre hunter legs'),
+					itemID('Vampyre hunter top'),
+					itemID('Vampyre hunter cuffs'),
+					itemID('Vampyre hunter hat'),
+					itemID('Vampyre hunter hat')
+				],
+				chance: [32, 32, 32, 32, 32]
+			},
+			{
+				item: [itemID('Vampyric plushie')],
+				chance: [250]
+			},
+			{
+				item: [itemID('Echo')],
+				chance: [1000]
+			}
+		]
+	}
+];
```

Comment:
I'm not sure im a fan of any of how this works. The thing I hate more than anything in code is having several sources of truth like this (e.g. if we change echo droprate, we now need to change it in 2 places and nobody will remember to do that in both).

For vladimir drakans case, you can use the original code just calling specialLoot. Also maybe do a quick check over all the other specialLoots to make sure nothing crazy will happen letting tames get that.

Reply:
At least for echo I could use the globalDroprates.ts rate. I'll try some more stuff out and see what I can come up with.

Reply:
You dont need too, just call monster.specialLoot(). Check monsterActivity